src/blas_like/level1/MaxAbs.cpp

```diff
@@ -11,46 +11,58 @@
 
 namespace El {
 
-    template<typename Real,typename>
-    Real MaxAbs( const Matrix<Real>& A )
+template<typename Real>
```

Comment:
This routine would work equally well with complex inputs as well (in which case the Elemental convention would be to change `Real` to `F`).